guides/source/action_view_overview.md

```diff
@@ -219,183 +205,137 @@ One way to use partials is to treat them as the equivalent of subroutines; a way
 <%= render "application/footer" %>
 ```
 
-Here, the `_ad_banner.html.erb` and `_footer.html.erb` partials could contain content that is shared among many pages in your application. You don't need to see the details of these sections when you're concentrating on a particular page.
-
-TIP: View partials rely on the same [Template
-Inheritance](/layouts_and_rendering.html#template-inheritance) as templates and
-layouts, so templates rendered by controllers that inherit from
-`ApplicationController` can render view partials declared in
-`app/views/application`.
+Here, the `_ad_banner.html.erb` and `_footer.html.erb` partials could contain content that is shared among many pages in your application. You don't need to see the details of these sections when you're focused on a Products' page.
 
-In addition to resolving partials with the inheritance chain, controllers can
-also override default partials with the inheritance chain. For example, a
-`ProductsController` that inherits from `ApplicationController` will resolve a
-call to `<%= render "ad_banner" %>` by first searching for
-`app/views/products/_ad_banner.html.erb` before falling back to
-`app/views/application/_ad_banner.html.erb`.
+The above example also uses the `_product.html.erb` partial. This partial contains details for rendering an individual product and is used to render each product in the collection `@products`.
 
-### `render` with `locals` Option
+### Passing Data to Partials With `locals` Option
 
-When rendering a partial, each key in the `locals:` option is available as a
-partial-local variable:
+When rendering a partial, we can pass data to the partial from the rendering view. We use the `locals:` options hash for this. Each key in the `locals:` option is available as a partial-local variable:
 
 ```html+erb
 <%# app/views/products/show.html.erb %>
 
-<%= render partial: "products/product", locals: { product: @product } %>
+<%= render partial: "product", locals: { my_product: @product } %>
 
 <%# app/views/products/_product.html.erb %>
 
-<%= tag.div id: dom_id(product) do %>
-  <h1><%= product.name %></h1>
+<%= tag.div id: dom_id(my_product) do %>
+  <h1><%= my_product.name %></h1>
 <% end %>
 ```
 
-If a template refers to a variable that isn't passed into the view as part of
-the `locals:` option, the template will raise an `ActionView::Template::Error`:
+A "partial-local variable" is a variable that is local to a given partial and only available from within that partial. In the above example, `my_product` is a partial-local variable. It was assigned the value of @product when passed to the partial from the original view.
+
+Note that typically we'd simply call this local variable `product`. We are using `my_product` to distinguish it from the instance variable name and template name.
+
+Since `locals` is a hash, you can pass in multiple variables as need like `locals: { my_product: @product, my_reviews: @reviews }`.
```

Comment:
```suggestion
Since `locals` is a hash, you can pass in multiple variables as needed, like `locals: { my_product: @product, my_reviews: @reviews }`.
```